cmd/root.go

```diff
@@ -47,11 +49,18 @@ var root = &cobra.Command{
 			return errors.New("could not get current user")
 		}
 
+		m, metaExists := meta.New().ParseHeader(slides[0])
+		if metaExists {
+			slides = slides[1:]
+		}
+
 		p := tea.NewProgram(model.Model{
-			Slides: slides,
-			Page:   0,
-			Author: user.Name,
-			Date:   time.Now().Format("2006-01-02"),
+			Slides:      slides,
+			Page:        0,
+			Author:      user.Name,
+			Date:        time.Now().Format("2006-01-02"),
+			Theme:       styles.SelectTheme(m.Theme),
+			CustomTheme: styles.CustomTheme(m.Theme),
```

Comment:
Is there a clever way to combine these two. My thinking was that we would put a `Renderer:     glamour.Renderer` on the model and handle the theme stuff before that.